index.html

```diff
@@ -2,240 +2,251 @@
+                              clicking on your name, 2) clicking the "Edit Profile" button,
+                              and 3) filling out/updating the "Pronouns" entry.
+                         </li>
+                    </ul>
+               </div>
           </div>
-     </div>
-     <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
-     <!-- Slack channels -->
-     <div class="item">
-          <div class="box">
-               <input type="checkbox">
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <!-- availability -->
+          <div class="item">
+               <div class="box">
+                    <input type="checkbox">
+               </div>
+               <div class="text">
+                    Consider setting your Slack status as active if you are available for synchronous activities. During
+                    the Sprint Day, you may set your status as <i>:hacking:</i> to advertise your availability for
+                    sprinting.
+               </div>
           </div>
-          <div class="text">
-               Please join the following Slack channels:
-               <ul>
-               <!-- collab-mtg channels -->
-               <li>
-                 <a href="https://lsstc.slack.com/archives/C90AMAG80"
-                   title="link to LSSTC Slack">#desc-collab-meeting</a>
-                    for announcements and daily briefings.
-               </li>
-               <li>
-                  <a href="https://lsstc.slack.com/archives/C0176US657B"
-                  title="link to LSSTC Slack">#desc-collab-meeting-hotline</a>
-                    to report any technical problems that may arise during the scheduled sessions on Monday to Thursday.
-               </li>
-               </ul>
+          <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+          <!-- Slack channels -->
+          <div class="item">
+               <div class="box">
+                    <input type="checkbox">
+               </div>
+               <div class="text">
+                    Please join the following Slack channels:
+                    <ul>
+                         <!-- sprint-week channels -->
+                         <li>
+                              <a href="https://lsstc.slack.com/archives/C2M8FG5A6"
+                                   title="link to LSSTC Slack">#desc-hack-sprint</a>
+                              for Sprint-related announcements.
+                         </li>
+                         <li>
+                              <a href="https://lsstc.slack.com/archives/C01EY4AK7UJ"
+                                   title="link to LSSTC Slack">#desc-sprint-hotline</a>
+                              to report any technical problems that may arise during Sprint Day.
+                         </li>
+                         <!-- <li>
+                              <a href="https://lsstc.slack.com/archives/C05NLP9DTK8"
```

Comment:
This should be removed or changed to a new channel.  I suspect that #desc-hack-sprint above may be enough

Reply:
I suspect that channel is too crowded for all the chatter related to local activities. I've created a new channel and linked here.